projectTemplates/ts-server/dockerfile

```diff
@@ -5,13 +5,18 @@ RUN npm install
 COPY . .
 RUN npm run build
 
-
-FROM node:12.18.3-slim as production
+FROM node:12.18.3-slim as node-base
 RUN apt-get update && apt-get -y install wget
+RUN mkdir /confd
+RUN wget -O '/confd/confd' 'https://github.com/kelseyhightower/confd/releases/download/v0.15.0/confd-0.15.0-linux-amd64'
+
+
+FROM node-base as production
 ARG NODE_ENV=production
 ENV NODE_ENV=${NODE_ENV}
 ENV SERVER_PORT=80
 WORKDIR /usr/app
+COPY /confd/confd ./confd
```

Comment:
If you are downloading confd in the base image, why do you need to copy it from outside?
In addition because of that copy its impossible to build the image without running confd outside.

Reply:
is a bug.
it should be copied from inside.
I updated the file